src/gwt/src/org/rstudio/studio/client/workbench/prefs/views/LineEndingsSelectWidget.java

```diff
@@ -22,11 +22,12 @@
 
 public class LineEndingsSelectWidget extends SelectWidget
 {
+   // TODO: i18n
```

Comment:
Is this able to be i18n'd?
![image](https://user-images.githubusercontent.com/14057112/124605993-7659d400-de3a-11eb-8b80-b02051ed53c1.png)

Reply:
I think the drop down vals are pulled through the `line_ending_conversion` obj in `user-prefs-schema.json` (within the cpp dir). It looks like Andrew did some of the other drop down vals this way. This could be the same for the `ChooseEncodingDialog` below

Reply:
See [this comment](https://github.com/ca-scribner/rstudio/pull/6#issuecomment-878196100) which I _think_ applies to the dropdown vals here, but the "Line ending conversion" label is hardcoded in the code, so should be able to i18n it

Reply:
Scratch that. Upon further investigation this one is a bit trickier as Andrew mentioned [here](https://github.com/StatCan/daaas/issues/510). We can tackle it in a separate issue.